core/src/main/java/io/questdb/PropertyKey.java

```diff
@@ -35,7 +35,6 @@ public enum PropertyKey {
     CAIRO_SNAPSHOT_INSTANCE_ID("cairo.snapshot.instance.id"),
     CAIRO_SNAPSHOT_RECOVERY_ENABLED("cairo.snapshot.recovery.enabled"),
     CAIRO_MKDIR_MODE("cairo.mkdir.mode"),
-    CAIRO_SQL_APPEND_PAGE_SIZE("cairo.sql.append.page.size"),
```

Comment:
What if someone has it in config, will QuestDB fail to start? As far as I remember we are not strict on config keys

Reply:
Yes, it would fail to start because of the `config.validation.strict=true` default. I've marked the `cairo.sql.append.page.size` prop as obsolete, so that users would get a more meaningful message in stdout. See 5f9cb513ceb8aba82a4ede2fef25717f92752e1d

Alternatively, we could mark it as deprecated, but then we should read the value of both old and new props. WDYT?

Reply:
Sorry, I was out of the loop for few days. Why are we making this prop obsolete? PR description should be present. Otherwise this looks like an insider knowledge

Reply:
> Why are we making this prop obsolete?

This property is unused.

Reply:
Updated the PR description.